chessence-frontend/src/pages/ChessBoard/ChessBoard.js

```diff
@@ -34,7 +34,8 @@ export default function Game(props) {
         };
     }, []);
 
-    const height_string = windowDimension.winHeight - 63.5 + "px"; // Convert to a string with 'px' appended
+    const height_string = windowDimension.winHeight - 110 + "px"; // Convert to a string with 'px' appended
+    console.log(height_string);
```

Comment:
do we want to console.log this value in `main`?

Reply:
The console.log() was for testing, I'll delete it.